### ISSUE TITLE
Interpreting Evidence Ratios

### ISSUE THREAD

**Issue by abigailkeller** [QERM514/assistance#35]
I just wanted to make sure that I am interpreting evidence ratios correctly. The model with the greatest weight would produce an evidence ratio of 1, and models with increasingly lower weights would produce increasingly larger evidence ratios. For example, if you had a model x with an evidence ratio of 4, would you interpret that as "model x has a 1 in 4 chance that it is more parsimonious than the top -ranked model" or "the top-ranked model is 4 times more likely than model x of being the best model"? Or are neither of these interpretations correct?

**Comment by mdscheuerell** [comment]
Yes, @abigailkeller, you are on the right track here. Perhaps an easier way to interpret the ER's is the evidence *against* the lower ranked (higher AIC) model having produced the data in hand. So, for example, an ER of 4 means there is a 1 in 4 chance the lower ranked model would have produced the data in your sample.

**Comment by abigailkeller** [comment]
Thanks!

**Comment by hmcmonagle** [comment]
This was helpful, thanks! 

How would we interpret ER = 1 for the best model (lowest delta AIC)? When the ER is 1 (for our best model), it doesn't make much sense that there is a "1 in 1 chance" (100% chance) that this model would have produced the data. So should we just interpret this as the best model has the lowest ER and therefore is the best supported by the data?

**Comment by mdscheuerell** [comment]
Yes, @hmcmonagle, that's correct.
